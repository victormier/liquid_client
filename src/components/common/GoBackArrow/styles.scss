@import 'base/variables';

.container {
  display: block;
  position: fixed;
  top: $base-grid-gutter;
  left: $base-grid-gutter;
  z-index: 10;
}
