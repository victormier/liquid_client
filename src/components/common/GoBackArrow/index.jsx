import React from 'react';
import arrowLeft from 'assets/images/arrowLeft.svg';
import { browserHistory } from 'react-router';
import styles from './styles.scss';

const GoBackArrow = () => (
  <div className={styles.container}>
    <a href="#back" onClick={browserHistory.goBack}>
      <img alt="arrow left" src={arrowLeft} />
    </a>
  </div>
);


export default GoBackArrow;
