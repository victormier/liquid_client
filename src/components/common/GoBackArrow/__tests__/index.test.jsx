import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import GoBackArrow from '../index';

describe('component', () => {
  it('renders', () => {
    const container = shallow(
      <GoBackArrow />
    );

    expect(shallowToJson(container)).toMatchSnapshot();
  });
});
