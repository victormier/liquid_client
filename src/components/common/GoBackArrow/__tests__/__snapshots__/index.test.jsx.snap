// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<div>
  <a
    href="#back"
    onClick={[Function]}
  >
    <img
      alt="arrow left"
      src={Object {}}
    />
  </a>
</div>
`;
